src/modules/pulseaudio.cpp

```diff
@@ -106,6 +120,17 @@ namespace modules {
   }
 
   string pulseaudio_module::get_output() {
+    // Exclude monitors and auto_null
+    if (!m_pulseaudio) {
+      return ""s;
+    }
+    auto s_name = m_pulseaudio->get_name();
+    if (s_name.empty() ||
+        s_name == "auto_null" ||
+        (s_name.size() >= 8 && s_name.compare(s_name.size() - 8, s_name.size(), ".monitor") == 0)) {
```

Comment:
Consider adding `ends_with` in [`string_util`](https://github.com/polybar/polybar/blob/master/include/utils/string.hpp#L64) and use it here.

Reply:
Thanks. I would like to get an ack about the big picture first. Once it's OK, I'll do what you suggest.